### PR TITLE
Remove footer part

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | comma-separated list of tickets fixed by the PR, if any
| License       | MIT

This part of the footer is useless 

After:

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/views/themes/material/footer.html.twig [hunk 1]
```diff
@@ -1,37 +1,4 @@
 <footer class="page-footer cyan darken-2">
-    <div class="container">
-        <div class="row">
-            <div class="col l6 s12">
-                <h5 class="white-text">{{ 'footer.wallabag.elsewhere'|trans }}</h5>
-                <p class="grey-text text-lighten-4">
-                    <a target="_blank" class="grey-text text-lighten-3" href="{{ addonsUrl.google_play }}" title="Android">
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/footer.html.twig]: Do we still need `addonsUrl` & `addonsUrl` to injected as global variable?
Baggy doesn't seems to use them.

Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/footer.html.twig]: I even this `FooterController` can be removed :)

Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/footer.html.twig]: Done.

**Comment by j0k3r** [comment]
:+1:

**Pull request merged**